services/openshift/openshift.service

```diff
@@ -14,7 +14,6 @@ ExecStartPre=-/usr/bin/docker rm openshift
 ExecStartPre=-/usr/bin/docker tag ${DOCKER_REGISTRY}/${IMAGE} openshift
 ExecStartPre=-/usr/bin/sh /opt/adb/openshift/docker_config $OPTIONS
 ExecStart=/usr/bin/sh /opt/adb/openshift/openshift $OPTIONS
-ExecStartPost=/bin/sleep 30
```

Comment:
+1 for removing this. This timeout might or might not work based on your machine. We need to actively check until openshift is up